assets/css/mform.css

```
@@ -441,115 +584,42 @@ body.rex-theme-dark .mform .custom-link.input-group.is-empty input.form-control[
 
-
-.rex-theme-light section.repeater .repeater-group,
 section.repeater .repeater-group {
     padding: 0.5rem 1.25rem 1.25rem 1.25rem;
     margin-bottom: 0.5rem;
     transition: box-shadow 0.3s ease-in-out;
 
-    border: 1px solid rgba(100, 100, 100, 0.20);
-    background: rgba(255, 255, 255, 0.09);
-    box-shadow: 0 5px 15px rgba(0, 0, 0, .08);
+    border: 1px solid var(--mform-repeater-group-border);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)